app/assets/src/components/views/discovery/discovery_view.scss

```diff
@@ -86,3 +86,24 @@
     max-height: 400px;
   }
 }
+
+.noDataBannerFlexContainer {
+  flex: 1 0 auto;
+  display: flex;
+  flex-direction: column;
+
+  &::before {
+    content: "";
+    flex: 1 0 auto;
+    max-height: 250px;
+  }
+
+  &::after {
+    content: "";
+    flex: 1 0 auto;
+  }
```

Comment:
does this vertically center the banner with a min top of 250px?  not that obvious to me, so maybe comment in the code

Reply:
yes

Reply:
Done